src/app/Nonagent/main.py

```diff
@@ -0,0 +1,100 @@
+from utils import make_request
+
+
+async def version(vicibox_server_ip: str) -> dict:
+    """
+    shows version and build of the API, along with the date/time and timezone
+
+    :param vicibox_server_ip:
+    :return:
+    """
+    url = f'http://{vicibox_server_ip}/vicidial/non_agent_api.php?function=version'
+    version_response = await make_request(url)
+    return version_response
+
+
+async def sounds_list(base_url: str, format: str = None, stage: str = None, comments: str = None) -> dict:
+    """
+    outputs a list of audio files from the audio store
+
+    :param base_url:
+    :param format:
+    :param stage:
+    :param comments:
+    :return:
+    """
+    function_url = 'function=sounds_list'
+
+    # Check if optional fields are present and add them to function url
+    if format:
+        function_url += f'&format={format}'
+    if stage:
+        function_url += f'&format={stage}'
+    if comments:
+        function_url += f'&format={comments}'
+
+    sounds_list_response = await make_request(f'{base_url}&{function_url}')
```

Comment:
Can we maybe use `create_base_non_agent_url` or create a similar util method to create the target URL here? I just think it's better if we take our URLs from one place for all methods in `main.py`, instead of defining it in `version` and then manually building it here. What do you think?